internal/k8shandler/cluster.go

```diff
@@ -181,6 +181,9 @@ func (er *ElasticsearchRequest) CreateOrUpdateElasticsearchCluster() error {
 		// add alias to old indices if they exist and don't have one
 		// this should be removed after one release...
 		if er.ClusterReady() {
+			// check if nodes are below watermark threshold and unblock indices if it's marked as read only
+			er.checkWatermarkAndUnblockIndices()
```

Comment:
Is this intended to be part of this change?

Reply:
i may need to rebase on top of master to clear this change. it should have been merged in already